tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -52,76 +46,63 @@ public function add_settings() {
 		$setting_helper = new Tribe__Extension__Settings_Helper();
 
 		$fields = array(
-			$this->opts_prefix . 'heading' => array(
+			$this->opts_prefix . 'heading'        => array(
 				'type' => 'html',
 				'html' => '<h3>' . esc_html__( 'Eventbrite Additional Options', 'tribe-extension' ) . '</h3>',
 			),
-			$this->opts_prefix . 'helper_text' => array(
-				'type' => 'html',
-				'html' => '<p>' . esc_html__( 'The following fields are from an Extension. These additional Eventbrite options allow you to change how the Eventbrite Tickets add-on operates, such as moving the ticket location on Single Event view, displaying tickets for Private Eventbrite events, and more.', 'tribe-extension' ) . '</p>',
-			),
-
 			$this->opts_prefix . 'content_before' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( '<h3> text to display Before/Above iframe', 'tribe-extension' ),
-				'tooltip'			=> esc_html__( 'By default there is no content before the iframe. You might want to try something like "Eventbrite Tickets". It will be output as an <h3> heading with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
-				'validation_type'	=> 'textarea', // runs wp_kses()
+				'type'            => 'text',
+				'label'           => esc_html__( 'Heading above ticket area', 'tribe-extension' ),
+				'tooltip'         => esc_html__( 'Adds a heading above the ticket area. By default, this is blank and no heading is shown.', 'tribe-extension' ),
+				'validation_type' => 'textarea',
 			),
 
 			$this->opts_prefix . 'content_after' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( '<span> text to display After/Below iframe', 'tribe-extension' ),
-				'tooltip'			=> esc_html__( 'By default there is no content after the iframe. It will be output as a <span> with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
-				'validation_type'	=> 'textarea', // runs wp_kses()
+				'type'            => 'text',
+				'label'           => esc_html__( 'Text underneath ticket area', 'tribe-extension' ),
+				'tooltip'         => esc_html__( 'By default, there is no content after the ticket area.', 'tribe-extension' ),
+				'validation_type' => 'textarea',
 			),
 
-			// see https://central.tri.be/issues/62402
 			$this->opts_prefix . 'iframe_px' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
-				'default'			=> 200, // required else positive_int validation will throw an error each time settings are saved if this field is left blank
-				'tooltip'			=> esc_html__( 'The default is 200.', 'tribe-extension' ),
-				'validation_type'	=> 'positive_int',
+				'type'            => 'text',
+				'label'           => esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
+				'default'         => 200,
+				'tooltip'         => esc_html__( 'The default is 200.', 'tribe-extension' ),
```

Comment:
Ordinarily we do not mention the default setting. There is no real need to, it visually complicates the settings area. But, it would make sense to mention here that the unit of measurement is pixels. Maybe this could `The height of the iframe ticket area in pixels.